code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -686,11 +686,11 @@
 	item_state = "DutchJacket"
 	body_parts_covered = ARMS
 
-/obj/item/clothing/suit/gorlex
+/obj/item/clothing/suit/ngr_captain
```

Comment:
could you please repath this under armor, or at least set the allowed var to the security vest's so you can actually hold weapons?

Reply:
I'm gonna repath it to capcarapace for consistency with the other captain jackets, but should I buff its armor? Right now even the Cybersun haori has normal carapace armor.